### PR TITLE
Re-enable previously failing test

### PR DESCRIPTION
**Pull request by aelred**
I'm going to be super picky and ask for 10 passes with this one

### CODE REVIEW

**Comment by aelred** [comment]
please retest this please - failed due to http://10.0.1.19:9998/job/unit-test-pull-request-grakn-titan/ai.grakn$grakn-test/3545/testReport/junit/ai.grakn.test.engine.controller/GraqlControllerTest/GETGraqlAggregateWithTextType_ResponseIsCorrect/

**Comment by aelred** [comment]
ok this test still fails :(

**Comment by aelred** [comment]
wwtd? !rtg

**Comment by aelred** [comment]
!rtg

**Comment by aelred** [comment]
one pass

**Comment by aelred** [comment]
@alexandraorth uncommenting the test may have made another test fail? 👍

**Comment by burukuru** [comment]
!rtg

**Comment by aelred** [comment]
!rtg - damn titan thingy

**Comment by aelred** [comment]
!rtg - one pass 👍

**Comment by aelred** [comment]
!rtg - two passes 👍 👍

**Comment by aelred** [comment]
!rtg - `TitanPreviousPropertyStepTest.whenUsingATitanGraph_ApplyStrategy`

**Comment by aelred** [comment]
!rtg - not sure what went wrong: https://jenkins.grakn.ai/job/unit-test-pull-request-grakn-titan/3987/

**Comment by aelred** [comment]
!rtg - three passes 👍  👍  👍

**Comment by aelred** [comment]
!rtg - four passes 👍 👍 👍 👍

**Comment by aelred** [comment]
!rtg - cassandra time-out in `grakn-dist`

**Comment by aelred** [comment]
!rtg - five passes 👍 👍 👍 👍 👍

File: grakn-test/src/test/java/ai/grakn/test/engine/GraknEngineServerIT.java [hunk 1]
```diff
@@ -98,8 +97,7 @@ public void whenSendingTasksToTwoEngines_TheyAllComplete(
         assertEquals(completableTasks(allTasks), completedTasks());
     }
 
-    @Ignore  // TODO: Fix this test - may be a race condition
-    @Property(trials=10)
```
Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/test/engine/GraknEngineServerIT.java]: I can't remember if I deliberate removed the `(trials=10)` - can you remember @alexandraorth ?

Comment (alexandraorth) [inline grakn-test/src/test/java/ai/grakn/test/engine/GraknEngineServerIT.java]: I think it was a side effect of removing the ignore- so not deliberate.

**Comment by aelred** [comment]
!rtg - six passes 👍 👍 👍 👍 👍 👍

**Comment by aelred** [comment]
!rtg - failure on ai.grakn.test.engine.controller.CommitLogControllerTest.whenCommittingGraph_TaskManagerReceivesCountTask

**Comment by aelred** [comment]
!rtg - seven passes 👍 👍 👍 👍 👍 👍 👍

**Comment by aelred** [comment]
!rtg - cassandra timeout issue

**Comment by aelred** [comment]
!rtg - eight passes 👍 👍 👍 👍 👍 👍 👍 👍

**Comment by aelred** [comment]
!rtg - nine passes 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Comment by aelred** [comment]
@alexandraorth strange failure. please investigate this thingummybob

**Comment by aelred** [comment]
!rtg

**Comment by aelred** [comment]
!rtg - cassandra timeouts (logged in TP)

**Comment by aelred** [comment]
!rtg - more cassandra timeouts

**Comment by aelred** [comment]
!rtg - `CommitLogControllerTest.whenCommittingGraph_TaskManagerReceivesCountTask` which there is a PR to ignore

**Comment by aelred** [comment]
TEN PASSES 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Comment by aelred** [comment]
@alexandraorth @fppt pls review

**Pull request merged**